keanu-project/src/main/java/io/improbable/keanu/network/ModelComposition.java

```diff
@@ -0,0 +1,98 @@
+package io.improbable.keanu.network;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import io.improbable.keanu.vertices.ProxyVertex;
+import io.improbable.keanu.vertices.Vertex;
+import io.improbable.keanu.vertices.VertexId;
+import io.improbable.keanu.vertices.VertexLabel;
+
+public final class ModelComposition {
+
+    private ModelComposition() { }
+
+    /**
+     * Compose one model within a wider model.
```

Comment:
Might be clearer to say connects one model to another, as the `compose within` makes it sound like it's compacted into one vertex